src/time_tool.cc

```diff
@@ -85,13 +87,20 @@ bool ParseCivilSpec(const std::string& args, cctz::civil_second* when) {
 
 const char* WeekDayName(cctz::weekday wd) {
   switch (wd) {
-    case cctz::weekday::monday: return "Mon";
-    case cctz::weekday::tuesday: return "Tue";
-    case cctz::weekday::wednesday: return "Wed";
-    case cctz::weekday::thursday: return "Thu";
-    case cctz::weekday::friday: return "Fri";
-    case cctz::weekday::saturday: return "Sat";
-    case cctz::weekday::sunday: return "Sun";
+    case cctz::weekday::monday:
```

Comment:
Let's drop all the other whitespace changes, unless you think the edit actually fixes something.